squarelet/users/constants.py

```diff
@@ -0,0 +1,4 @@
+# Standard Library
+from datetime import timedelta
+
+MFA_PROMPT_SNOOZE_DURATION = timedelta(weeks=2)
```

Comment:
I would make this a setting in `settings.py` so it can be updated via environment variables